src/components/menu/menu.e2e.ts

```diff
@@ -0,0 +1,7 @@
+import { hidden, renders } from "../../tests/commonTests";
+
+describe("calcite-menu", () => {
+  it("renders", async () => renders("calcite-menu", { display: "flex" }));
+
+  it("honors hidden attribute", async () => hidden("calcite-menu"));
```

Comment:
Do we want to try adding a11y tests at this point or wait for that?

Reply:
added a11y for `menu-item` and `menu` components.